src/main/scala/beam/agentsim/agents/rideHail/RideHailingManager.scala

```diff
@@ -859,14 +859,12 @@ class RideHailingManager(
     }
   }
 
-  def lockVehicle(vehicleId: Id[Vehicle]) = {
+  def lockVehicle(vehicleId: Id[Vehicle]): Unit = {
     lockedVehicles += vehicleId
   }
 
 
-  def getVehicleFuelLevel(vehicleId: Id[Vehicle]): Double = {
-    vehicleFuelLevel.get(vehicleId).get
-  }
+  def getVehicleFuelLevel(vehicleId: Id[Vehicle]): Double = vehicleFuelLevel(vehicleId)
```

Comment:
Should this return an Option? Granted this is not really used afaict. Just as an API example in `StanfordRideHailAllocationManagerV1`, so probably not a big deal...maybe even remove?